src/views/RecIM/views/Home/index.js

```diff
@@ -13,6 +13,14 @@ import { getParticipantTeams, getParticipantByUsername } from 'services/recim/pa
 import WaiverForm from 'views/RecIM/components/Forms/WaiverForm';
 import CreateSeriesForm from 'views/RecIM/components/Forms/CreateSeriesForm';
 
+const TabPanel = ({ children, value, index }) => {
+  return (
+    <div hidden={value !== index} role="tabpanel">
```

Comment:
I don't like this really, why do you return a hidden div when you could just check with Javascript with a conditional return.

Reply:
That's a good point, thank you for clearing that up. It won't refetch any items but it will make the children load slower.